### ISSUE TITLE
nodeos_startup_catchup_lr_test seems to be flakey

### ISSUE THREAD

**Issue by NorseGaud** [EOSIO/eos#7842]
EOSIO LRT `nodeos_startup_catchup_lr_test` failing:

Current pipeline: https://buildkite.com/EOSIO/eosio/builds/16411#a9af212e-03a4-418b-b90d-652de454956c
```
error 2019-09-03T13:28:23.184 keosd     http_plugin.cpp:678           handle_exception     ] FC Exception encountered while processing wallet.import_key
```

We see this in the new pipeline too with separate parallelized testing: 
https://buildkite.com/EOSIO/eosio-beta/builds/1761#aaad5d94-3ace-4bff-995f-a1980f7f8498
https://buildkite.com/EOSIO/eosio-beta/builds/1743#129420b3-fb85-4683-8f8a-1d57a929e373
https://buildkite.com/EOSIO/eosio-beta/builds/1759#b5aa9dbc-edb4-43cc-97a9-38f518aa1f00
https://buildkite.com/EOSIO/eosio-beta/builds/1761#9f62dedb-29bf-4785-b6ba-4a90e2b231d4

```
AssertionError: Expected terminating "programs/nodeos/nodeos --max-transaction-time -1 --abi-serializer-max-time-ms 990000 --filter-on "*" --p2p-max-nodes-per-host 13 --contracts-console --plugin eosio::producer_api_plugin  --config-dir etc/eosio/node_08 --data-dir var/lib/node_08 --genesis-json etc/eosio/node_08/genesis.json --genesis-timestamp 2019-08-30T22:05:28.894

" to have an exit status of 0, but got -11
```

Retrying usually works.

Macs are running 6 cores and 14GB.

- We've tried running with 12 cores and 24G of memory with no luck. It looks like it just takes half the time to fail (54m vs ~30m): https://buildkite.com/EOSIO/eosio-beta/builds/1798#81b0df67-df70-4fa3-9068-c58d3a775348
- Unable to replicate it locally on my mac
- We also tried running it without splitting it up: https://buildkite.com/EOSIO/eosio-beta/builds/1795#bf1c7156-449d-4a0b-ad4a-9e34bbde30ae

Linux is running 7 cores and 25G.

- Harder to replicate linux failures.
- Fails running through docker locally

**Comment by NorseGaud** [comment]
Additionally, we used to have linux builders running 12 cores. We had to change that to 7 so we could fit two on the same container and get more agents. However, mac has always been using 6. It's almost as if we see more of this problem since we moved LRTs to be split up (parallelized) in the new pipeline.

**Comment by heifner** [comment]
Fixed by #7860

**Comment by heifner** [comment]
Ran every 4hrs over the weekend with on failures.
